configuration/wireless/headless.md

```diff
@@ -30,3 +33,18 @@ SSH can be enabled by placing a file called `ssh` in to the boot folder. This fl
 
 See [here](../../remote-access/ssh/README.md#3-enable-ssh-on-a-headless-raspberry-pi-add-file-to-sd-card-on-another-machine) for more details.
 
+## Enabling VNC
+Once the SSH and wpa_supplicant files are in place you will need do download putty in your PC and first you will also need to identify the IP of your device via router stats or if its connected to a phone you can directly check its IP. 
+
+After getting the IP you need to enable Real VNC. For that download Putty and put the IP of your Raspberry Pi and click on ssh and click open. Now your device is connected via SSH. Now you need to configure it inorder to see its screen on your device.
+
+When asked for user ID and Password type user ID : pi pswd:raspberry (If you have changed it to something else type that)
+Now type 
+sudo raspi-config
+
+Now the Configuration menu will open now go to Interfacing options--> RealVNC--> Enable
+Restart your PI after it.
+
+Now just put the IP of your raspberry Pi on your VNC viewer and you're good to go.
+
+If you want to play games don't forget to enable direct capture in VNC settings otherwise it will not show you the app on the screen.
```

Comment:
Enabling VNC does seem to be a thing that people running headless might want to do, just not sure this is the right way to go about it. Will have a think.